app/javascript/app/pages/ghg-emissions/ghg-emissions-component.jsx

```diff
@@ -29,7 +29,10 @@ class GhgEmissions extends PureComponent {
         {!isPageContained && (
           <Header route={route}>
             <div className={cx(layout.content, styles.header)}>
-              <Intro title="Historical GHG Emissions" />
+              <Intro
+                title="Historical GHG Emissions"
+                description="Climate change causing greenhouse gas emissions have increased 50 fold since the late 1800s. Energy makes up nearly two-thirds of global emissions, followed by agriculture. Within the energy sector, electricity and heat generation make up the largest portion of emissions, followed by transportation and manufacturing. In 2014, 60% of global greenhouse gas emissions came from just 10 countries, while the 100 least-emitting contributed less than 3%. "
```

Comment:
Thank you for adding this. There are some links and we would have to check where they go to

Reply:
We need to how all these changes. The dropdown and the text only if we have FEATURE_NEW_GHG=true. Sorry for the late notice. We have been deploying features to production and we want to be more strict about this

Reply:
Are you referring to the buttons? I mean there should be two versions of the `buttonGroup`? One with the old set up and one with the new?

**new**:
![image](https://user-images.githubusercontent.com/6906348/80218789-eb7c2780-8641-11ea-9568-1f3efdfe9be2.png)

**old**:
![image](https://user-images.githubusercontent.com/6906348/80218870-05b60580-8642-11ea-99a4-8c19fdf82ddf.png)


If that's the case wouldn't it be better to have a separated branch and not merge it to `develop`?

Reply:
I was referring to all the changes. I'm sorry for the late notice but we don't want to show the changes directly on production. As for the buttons is ok if we keep the share button, but the download should show as a button on production. 
I prefer to handle this with a feature flag to avoid conflicts between the different tasks and easier to deploy when we need it. We already have the FEATURE_NEW_GHG for this. If you feel that this is not the best option we can discuss this.

Reply:
Ok. Thanks for the clarifications. I'll add the `csv` download and the `data explorer download` to the button group then.